README.md

```diff
@@ -276,6 +304,10 @@ Compose.prototype.map = function(f) {
 
 #### `sequence` method
 
+```hs
+sequence :: Apply f, Traversable t => t (f a) ~> (b -> f b) -> f (t a)
```

Comment:
Ah, so you're drawing a distinction between a type which *could* provide `of` and a type which *actually* provides it. My view is that every type which can provide `of` should do so, but I understand your position.